data/yaml/venues.yaml

```diff
@@ -486,6 +486,9 @@ knlp:
   acronym: knlp
   name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge and Neural
     Networks for NLP'
+konvens:
+  acronym: KONVENS
+  name: Conference on Natural Language Processing
```

Comment:
Can you add a `url: https://konvens.org/site/` field?

Reply:
done